ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/SearchFilterKeywordCriteria.cpp

```diff
@@ -35,20 +35,20 @@ QString highlightedMidpoint(const QString& string,
 
     // Find the portion of string that goes before the catchment.
     int startPos = std::max(0, pos-catchment);
```

Comment:
Maybe this variable can be `const` too. And more variables.

Is there a guideline about the usage of `const` in samples?